.changeset/giant-moles-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@spectrum-web-components/overlay': minor
+---
+
+prevent overlay close on document scroll
```

Comment:
Should this include more details and be formatted since this is the content that will go into the changelog when published? Do we have any best practices about how SWC likes the changelog content to be structured? Maybe it should include the Jira ticket info as well?

Reply:
I am working on a branch to surface up a global formatted changelog. Would like to keep the same that we were doing with lerna. Please let me know if you have any ideas on this. Would love to know!

Reply:
We note down all the migration of breaking changes [here](https://opensource.adobe.com/spectrum-web-components/migrations/2024-10-31%20(1.0.0)/)

Reply:
@Rajdeepc see these recommendations for writing changesets: https://github.com/RedHat-UX/red-hat-design-system/wiki/Changesets